app/models/foreman_docker/docker.rb

```diff
@@ -131,17 +132,19 @@ def bootstrap(args)
     end
 
     def client
-      @client ||= ::Fog::Compute.new(
+      cli = ::Fog::Compute.new(
           :provider         => 'fogdocker',
           :docker_username  => user,
           :docker_password  => password,
           :docker_email     => email,
           :docker_url       => url
       )
+      cli.api_version.present?
```

Comment:
that was done more as an intent to do a ping test to ensure that the connection worked..